src/Storage/Services/IRegisterService.cs

```diff
@@ -18,9 +18,8 @@ public interface IRegisterService
         /// <summary>
         /// Party lookup
         /// </summary>
-        /// <param name="orgNo">organisation number</param>
-        /// <param name="person">f or d number</param>
-        /// <returns></returns>
-        Task<int> PartyLookup(string orgNo, string person);
+        /// <param name="person">The f or d number to look up</param>
+        /// <param name="orgNo">The organisation number to look up</param>
+        Task<int> PartyLookup(string person, string orgNo);
```

Comment:
would make more sense here to populate the returns statement rather than removing it all together. 

`<returns>>The partyId corresponding to the provided person or organisation</returns>`